doc/changes/0.1.inc

```diff
@@ -0,0 +1,18 @@
+.. _changes_0_1:
+
+Version 0.1 (2011-09-23)
+------------------------
+
+- Initial release (initial commit: 2010-12-26).
```

Comment:
note the xmas period me arriving in Boston late 2010 :)

Reply:
Yes, I had a short laugh about these dates - there is even one on christmas eve :heart_eyes: